fedora/client/bodhi.py

```diff
@@ -17,6 +17,10 @@
 # License along with python-fedora; if not, see <http://www.gnu.org/licenses/>
 #
 """
+DEPRECATED: This module is deprecated and has moved to bodhi.client.bindings.
+As the bodhi-0.9 client still uses it, it will remain here until Fedora 25 is
+EOL.
+
```

Comment:
@pypingou I considered that, but I don't think I'd like the bodhi client in F24/F25 to be emitting deprecation warnings if I can help it. I think what I'd like to do is to stick with this message for now, then once F27 is released we can add in a deprecation warning, and then after n months we can delete this module. What do you think?